### PR TITLE
New Presence - Online Sequencer (https://onlinesequencer.net/)

### PR DESCRIPTION
**Pull request by YJJcoolcool**

### CODE REVIEW

**Comment by YJJcoolcool** [comment]
Closed Pull Request due to errors

**Comment by ririxidev** [comment]
just fix them there?

**Comment by YJJcoolcool** [comment]
Ok, all issues fixed.

File: websites/O/Online-Sequencer/presence.ts [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+    // element grabs //
+    // api calls //
+    // variable sets //
+}
+
+setInterval(myOutsideHeavyLiftingFunction, 10000);
+//Run the function separate from the UpdateData event every 10 seconds to get and set the variables which UpdateData picks up
+
+*/
```
Comment (Bas950) [inline websites/O/Online-Sequencer/presence.ts:23]: remove comments

**Review by Bas950 — CHANGES_REQUESTED** [review]
upload proof of it working

**Comment by YJJcoolcool** [comment]
> upload proof of it working

Do I send like a screenshot or video of it working?

**Comment by Bas950** [comment]
> > upload proof of it working
> 
> Do I send like a screenshot or video of it working?

Just screenshot proof of how its shown in Discord.

**Comment by YJJcoolcool** [comment]
> > > upload proof of it working
> > 
> > 
> > Do I send like a screenshot or video of it working?
> 
> Just screenshot proof of how its shown in Discord.

Hi, I've sent the screenshots to the #presence-dev channel on the PreMiD discord. For your convenience, here's the link to the screenshots: https://imgur.com/a/fGZ9RZe

**Review by Bas950 — APPROVED** [review]
upload proof of it working

**Review by ririxidev — APPROVED** [review]

**Pull request merged**